src/openapi/streaming/serializer/wrappers/protobuf-wrappers.js

```diff
@@ -7,10 +7,34 @@ export default {
                 },
 
                 toObject(message, options) {
-                    const seconds = Number(message.seconds) * 1000;
-                    const date = new Date(seconds);
+                    const { seconds, nanos } = message;
+                    const MILIS_PER_SECOND = 1000;

```

Comment:
Maybe we can simplify it. Check google timestamp wrappere for JS:
https://github.com/google/protobuf/blob/master/src/google/protobuf/compiler/js/well_known_types/timestamp.js

Reply:
Good idea!